app-admin/graylog/graylog-3.3.8.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Free and open source log management"
+HOMEPAGE="https://www.graylog.org"
+SRC_URI="https://downloads.graylog.org/releases/graylog/${P}.tgz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+RESTRICT="strip"
+
+RDEPEND="!app-admin/graylog2
+	acct-group/graylog
+	acct-user/graylog
+	>=virtual/jdk-1.8:*"
```

Comment:
The correct way to declare a dependency on the JVM and have Portage get the JVM ready for an ebuild that might require it is as follows:
```
inherit java-pkg-opt-2

pkg_setup() {
        java-pkg-opt-2_pkg_setup
}

RDEPEND="
    >=virtual/jre-1.8
"

DEPEND="
    >=virtual/jdk-1.8
"
```

Reply:
Hello @monsieurp, actually we don't need Java for building itself, it's a binary package, so Java will be needed just for running which is expressed in the current RDEPEND.